telemetry/data_pipeline_mvp/grab_from_serial.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+"""
+objective - grab data from serial, parse, pass into redis timeseries 
+
+"""  
+
+import serial
+import time
+from redistimeseries.client import Client
+
+# figure out way to automatically detect port from python? 
+
+
+
+class Data_parser(object):
+    def __init__(self):
+        self.port_name = "/dev/ttyUSB0"
+        self.baudrate = 9600
+        self.read_frequency = 20 # in hz  
```

Comment:
Also, good note here that it is in Hz but that is something you should consider in variable naming. `self.read_frequency_hz` is a much more effective variable name than `self.read_frequency` with a comment denoting the unit